app/styles/components/_hero.scss

```diff
@@ -18,12 +19,22 @@ $hero-max-width: 600px;
   background-image: url('../images/experts@2x.jpg');
 }
 
+.Hero--team {
+  background-image: url('../images/officers-board@2x.png');
+}
+
 .Hero-title {
   margin-top: $Theme-margin-half;
 
   text-align: center;
 }
 
+.Hero-title--team {
```

Comment:
I think that in `SUIT CSS` we don't apply `modifiers` to `descendants`. In this case, we already have a modifier for the `Hero` components called `--team` so why not nest this rule inside it?  It also seems to me that you are creating   a new `descendant` unnecessarily. Why don't you do something like this:

```
.Hero.Hero--Team {
  background-image: url('../images/officers-board@2x.png');

  .Hero-title {
    margin-top: $team-margin-small;
  }
}
```